gh-badges/lib/make-badge.spec.js

```diff
@@ -94,7 +94,11 @@ describe('The badge generator', function() {
 
   describe('JSON', function() {
     it('should always produce the same JSON (unless we have changed something!)', function() {
-      const json = makeBadge({ text: ['cactus', 'grown'], format: 'json' })
+      const json = makeBadge({
+        text: ['cactus', 'grown'],
+        format: 'json',
+        links: ['https://example.com/', 'https://other.example.com/'],
```

Comment:
should this be `link:`, or is this one different on purpose?

Reply:
This one is different on purpose, as the plural form is used in [make-badge.js](https://github.com/badges/shields/blob/75def85eca5f045b147191e8142f3ce146051c4f/gh-badges/lib/make-badge.js#L107).

Reply:
We can fix that as part of #3370 😀